### PR TITLE
Implemented home page

### PR DESCRIPTION
**Pull request by carlows**
# Context

This PR implements the home page.

It also fixes the layout of the footer in tablet devices.

### Mobile

image

image

### Tablet

image

image

### Desktop

image

image

### General

http://recordit.co/aj5r5LjG8o (too big for github 🙂)

### Added opacity to the background icons in the services page

image

### CODE REVIEW

**Review by duranmla — CHANGES_REQUESTED** [review]
Same as @TheMightyPenguin let's not use includes for snippets that belongs only to a certain page, besides, make sure the name of the settings jekyll file match the format.

File: _sass/_layouts--global.scss [hunk 1]
```diff
@@ -6,13 +6,6 @@ body {
 .site {
   display: flex;
   flex-direction: column;
-
-  &-content {
-    .container:first-of-type,
-    .container-fluid:first-of-type {
-      padding-top: $size--header-height;
```
Comment (duranmla) [inline _sass/_layouts--global.scss:13]: @Carlows remember to add the padding top on services page

File: css/settings-jekyll.scss [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  purpose. If there's ever the need to include a jekyll variable on a
+  stylesheet, it'll be done on this file. It should be kept organized
+  with the same sections/components that are on the `_sass` folder.
+*/
+
+@import "../node_modules/bootstrap/scss/variables";
+@import "../node_modules/bootstrap/scss/mixins";
+
+/**
+ * HOME Lead section
```
Comment (TheMightyPenguin) [inline css/settings-jekyll.scss]: Below this section name,could you also put the name of the file where the class is defined? 
like

```
/**
 * Specific services sections
 * _sass/_components--service.scss
 */
```

File: _includes/head.html [hunk 3]
```diff
@@ -6,6 +6,7 @@
   <title>{% if page.title %}{{ page.title | escape }}{% else %}{{ site.title | escape }}{% endif %}</title>
   <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
 
+  <link rel="stylesheet" href="{{ "/css/settings-jekyll.css" | prepend: site.baseurl }}">
```
Comment (TheMightyPenguin) [inline _includes/head.html]: Could you include this after `main.css`?

**Review by TheMightyPenguin — CHANGES_REQUESTED** [review]

File: css/settings-jekyll.scss (hunk 2 above)
Comment (TheMightyPenguin) [reply css/settings-jekyll.scss]: Also, this file should be named `settings--jekyll.scss`

File: _sass/_partials--services.scss [hunk 4]
```diff
@@ -2,10 +2,16 @@
   left: 0;
   margin: 0 auto;
   max-width: 1600px;
+  opacity: 0.7;
+  padding-top: $size--header-height;
```
Comment (carlows) [inline _sass/_partials--services.scss:6]: @TheMightyPenguin it's here

**Review by TheMightyPenguin — APPROVED** [review]
LGTM :+1:

**Review by duranmla — APPROVED** [review]

**Pull request merged**